scripts/utility.py

```diff
@@ -1665,6 +1665,43 @@ def change_relationship_values(
                     if log_text not in rel.log:
                         rel.log.append(log_text)
 
+def get_kin_groups(cat, include_dead=True):
```

Comment:
Is it important at this point to already have the cat class? It would be more performant to just use the id's as long as possible and then fetch the cat, when the information from them is needed.